_posts/fr/2023-05-17-policy.md

```diff
@@ -63,7 +63,19 @@ h2:not(:first-of-type) { margin-top: 3em; }
 
 {% include specials/policy/fr/06-consistence.md %}
 
+## Ressources du réseau
+
+*Originally published in [Newsletter #257](/fr/newsletters/2023/06/28/#en-attente-de-confirmation-7--ressources-du-réseau)*
+
+{% include specials/policy/fr/07-ressources-du-reseau.md %}
+
+## La politique comme interface
+
+*Originally published in [Newsletter #258](/fr/newsletters/2023/07/05/#en-attente-de-confirmation-8--la-politique-comme-interface)*
```

Comment:
We have "Originally published in Newsletter..." for the policy entries already (sorry I missed this). But we should consider a follow up PR to translate those.